azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/arm/implementation/AzureConfigurableImpl.java

```diff
@@ -54,6 +55,19 @@ public T withInterceptor(Interceptor interceptor) {
         return (T) this;
     }
 
+    @SuppressWarnings("unchecked")
+    @Override
+    public T withAuxiliaryTokens(AzureTokenCredentials... tokens) {
```

Comment:
Should this be named `withAuxiliaryCredentials(AzureTokenCredentials... credentials)`?

Reply:
Fixed